### PR TITLE
Aug 2022 package update

### PR DESCRIPTION
**Pull request by Nikomus**
## Description

- Raise node version to v16 as Vercel are removing support for v12
- Update prettier and `yarn format`
- Update SVG packages (do we need three packages to supply six icons..?)

### CODE REVIEW

**Review by rosejbon — APPROVED** [review]
Looks `prettier` (I have made myself laugh with my own bad pun! 😆 ) 

:shipit:

File: package.json [hunk 1]
```diff
@@ -62,10 +62,10 @@
     "husky": "4.2.5",
     "jest": "25.5.4",
     "lint-staged": "12.5.0",
-    "prettier": "1.19.1"
+    "prettier": "2.7.1"
   },
   "engines": {
-    "node": "12.22.x",
+    "node": ">=16.15",
     "yarn": "1.22.19"
```
Comment (charlesstrange2) [inline package.json:69]: Will this roll to 18 when that happens?

**Review by charlesstrange2 — APPROVED** [review]
Tighten the node version, maybe?

File: package.json (hunk 1 above)
Comment (Nikomus) [reply package.json:69]: The version should stay as 16 in the Vercel settings, but wouldn't break if it went up. I'll look at if there's a benefit to it being more specific next time.

**Pull request merged**